.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "prettier.eslintIntegration": true
+}
```

Comment:
I can see this file in the `.gitignore`. I think you committed it before adding it to the `.gitignore`. Try deleting it and making a new commit.